IOPool/Input/test/testSchemaEvolution.sh

```diff
@@ -91,6 +91,20 @@ cmsRun ${LOCAL_TEST_DIR}/SchemaEvolution_test_read_cfg.py --inputFile "$inputfil
 
 file=SchemaEvolutionTestOLD13_0_0.root
 inputfile=$(edmFileInPath IOPool/Input/data/$file) || die "Failure edmFileInPath IOPool/Input/data/$file" $?
+
+# Known ROOT bug in 13_0_0 should cause this to fail
```

Comment:
Could you add more comments on the purpose of this test (that it is about the file read error exit code)? The purpose of this test is somewhat different from the other tests here (but maybe nevertheless a good-enough place for it)

Reply:
Done.